spec/services/user_invite_service_spec.rb

```diff
@@ -79,7 +79,7 @@
 
     user = User.find_by(email: "partner@example.com")
     expect(user.name).to eq("Partner User")
-    expect(user).not_to be_nil
+    expect(user).to be
     expect(user).to have_role(Role::PARTNER, @partner)
```

Comment:
This change doesn't seem necessary.  Though given the order of the checks, I don't think this line is ever going to fail anyway -- if the user doesn't exist, then user.name is certainly not going to equal "Partner User".